paddle/framework/program_desc.cc

```diff
@@ -64,5 +67,27 @@ ProgramDesc::ProgramDesc(const std::string &binary_str) {
   }
 }
 
+const std::vector<std::string> ProgramDesc::GetFeedVarNames() {
+  BlockDesc *global_block = blocks_[0].get();
+  std::vector<std::string> feed_var_names;
+  for (auto *op : global_block->AllOps()) {
+    if (op->Type() == "feed") {
+      feed_var_names.insert(feed_var_names.begin(), op->Output("Out")[0]);
+    }
+  }
+  return feed_var_names;
+}
+
+const std::vector<std::string> ProgramDesc::GetFetchVarNames() {
+  BlockDesc *global_block = blocks_[0].get();
+  std::vector<std::string> fetch_var_names;
+  for (auto *op : global_block->AllOps()) {
+    if (op->Type() == "fetch") {
```

Comment:
fetch -> kFetchOpType
and let's rename `fetch_var_names` to `fetch_target_names` if there is no better candidate.